lib/notify.js

```diff
@@ -0,0 +1,25 @@
+const { window, workspace, ConfigurationTarget } = require('vscode');
+const { EXTENSION_CONFIG_NAME, EXTENSION_CONFIG_KEYS } = require('./constants');
+
+const HAS_SEEN_BETA = 'HS_HAS_SEEN_BETA';
```

Comment:
Maybe `HAS _SEEN_BETA` can live in the constants file? Maybe in some `GLOBA_STATE_KEYS` object.

Reply:
Good idea!